app/controllers/accounts_controller.rb

```diff
@@ -78,6 +78,14 @@ def destroy
 
   private
 
+    def decide_layout
+      if can? :read, :admin_dashboard
```

Comment:
Hm, would we ever want to show the `accounts#new` action using the admin template? Or maybe this is where the admin workflow and normal end-user workflow ought to diverge.